node/Cargo.toml

```diff
@@ -57,6 +55,7 @@ prettyplease = "0.2.6"
 pretty_assertions = "1.4.0"
 quick-protobuf = "0.8.1"
 rand = "0.8.0"
+rand04 = "0.1.1"
```

Comment:
Why would you need to rely on a separate crate? You can rename `rand` v0.4 [using something like](https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html#renaming-dependencies-in-cargotoml)

```toml
rand04 = { package = "rand", version = "0.4" }
```

Reply:
I went for it because I thought it offers some compatibility, but forgot to revert. 

Fixed.